reports/report_details.py

```diff
@@ -1095,7 +1095,7 @@ def add_transnational_to_regions(regions):
         '2015': {
             'name': '71',
             'title': 'Key themes, women\'s overall presence',
-            'desc': '',
+            'desc': 'Key themes, women\'s overall presence',
```

Comment:
Queries `70` to `78 do` not have description in the queries sheet @ascii-dev and I removed them in the previous PR. Why are we adding them back?

Reply:
Must've been an issue with moving commits around from branch to branch. I've gone ahead to fix them though. @kilemensi